kernel/arch/dreamcast/sound/snd_sfxmgr.c

```diff
@@ -85,228 +85,273 @@ void snd_sfx_unload(sfxhnd_t idx) {
     free(t);
 }
 
-/* WAV header:
-    0x08    -- "WAVE"
-    0x14    -- 1 for PCM, 20 for ADPCM
-    0x16    -- short num channels (1/2)
-    0x18    -- long  HZ
-    0x22    -- short 8 or 16 (bits)
-    0x28    -- long  data length
-    0x2c    -- data start
-
- */
+/* WAV header */
+typedef struct __attribute__((__packed__)) {
```

Comment:
the attribute is not really useful here - looking at the fields, it will always be packed.

Reply:
It doesn't hurt to keep it though... I'm usually of the mindset that file headers and the like should always be declared as packed just in case the compiler decides to do silly things later.